assets/i18n/en_US.json

```diff
@@ -234,8 +236,12 @@
         "autoUpdatePatchesHint": "Automatically update patches to the latest version",
         "universalPatchesLabel": "Show universal patches",
         "universalPatchesHint": "Display all apps and universal patches (may slow down the app list)",
+
         "versionCompatibilityCheckLabel": "Version compatibility check",
         "versionCompatibilityCheckHint": "Restricts patches to supported app versions",
+        "requireSuggestedAppVersionLabel": "Require suggested app version",
+        "requireSuggestedAppVersionHint": "Enforce selection of suggested app version",
+        "requireSuggestedAppVersionDialogText": "Selecting an app that does not have the suggested version may cause unexpected issues.\n\nDisable anyways?",
```

Comment:
How about `Selecting an app that is not the suggested version may cause unexpected issues. Do you want to proceed anyways?`

Reply:
```suggestion
        "requireSuggestedAppVersionDialogText": "Selecting an app that is not the suggested version may cause unexpected issues.\n\nDo you want to proceed anyways?",
```